Microsoft.Toolkit.Uwp.UI/Extensions/TextBox/TextBoxExtensions.Mask.Internals.cs

```diff
@@ -297,13 +297,18 @@ private static void Textbox_TextChanging(TextBox textbox, TextBoxTextChangingEve
             // case adding data at the end of the textbox
             if (oldSelectionStart >= oldText.Length && !isDeleteOrBackspace)
             {
-                textbox.Text = textbox.Text.Substring(0, oldText.Length);
-                if (oldText.Length >= 0)
+                // ignore change(s) if oldtext is a substring of new text value
+                if (textbox.Text.Contains(oldText, StringComparison.OrdinalIgnoreCase))
```

Comment:
```suggestion
                if (textbox.Text.Contains(oldText))
```

Case changes matter here. So, don't ignore it. But if we need to ignore, then include a variable and make it an option that can be set.

 ```suggestion
                if (textbox.Text.Contains(oldText, compareMode))
```